src/meta/src/model/stream.rs

```diff
@@ -401,7 +411,27 @@ impl TableFragments {
         result
     }
 
+    pub fn update_table_fragment_map(&mut self, fragment_id: FragmentId) {
```

Comment:
When will this function be called? I think the `table_to_fragment_map` should be maintained automatically.

Reply:
Since fragment's internal state tables are set during scheduling, `TableFragment` can't be touched. This's a workaround to maintained `table_to_fragment_map`. I will try to refactor this when start refactoring DDL service.